### ISSUE TITLE
After writing image with TIffWriter.write output is black.

### ISSUE THREAD

**Issue by Yuvi-416** [cgohlke/tifffile#35]
Hi there,
I saw you commented there (https://github.com/python-pillow/Pillow/issues/5012), I was trying to post this question there but unfortunately, that issue was closed as a duplicate.
I once again went through a tifffile package and I found that I can use TiffWriter.write a function to write a TIFF file within a compressed of jpeg or JPEg2000.

1. I tried to compress image shape = (9152, 46624) with size 426.7mb.
1.1 At first I run this code with jpeg compression and I got and nice output of size 105.6 MB
img = tifffile.imread(input_C_10)
with TiffWriter(str(output_C_10)+'compressed_100.tif') as tif:
    tif.write(img, compression='jpeg')
1.2 secondly I run this code with Jpeg2000 compression and I got an output of 220.mb but the image is pull black.
img = tifffile.imread(input_C_10)
with TiffWriter(str(output_C_10)+'compressed_100.tif') as tif:
    tif.write(img, compression='jpeg2000')
as u can see here. It´s totally black.
![Screenshot from 2020-10-27 17-28-53](https://user-images.githubusercontent.com/65744819/97339076-0d1ccc80-187a-11eb-82d2-711689a685e0.png)

Secondly,
2. I tried to compress image shape = (36608, 186496) with size 6.83 Gb.
2.1. at first with jpeg compression and I got an error as follows:
Traceback (most recent call last):
  File "/home/yuvi/PycharmProjects/task-1/VIdeo_frame_extraction/NDPI-ext/TEMP-1-COMPRESSION.py", line 20, in <module>
    tif.write(img, compression='jpeg')
  File "/home/yuvi/anaconda3/envs/task-1/lib/python3.7/site-packages/tifffile/tifffile.py", line 2340, in write
    strip = compress(strip)
  File "/home/yuvi/anaconda3/envs/task-1/lib/python3.7/site-packages/imagecodecs/imagecodecs.py", line 829, in jpeg_encode
    optimize=optimize, smoothing=smoothing, out=out)
  File "imagecodecs/_jpeg8.pyx", line 211, in imagecodecs._jpeg8.jpeg8_encode
imagecodecs._jpeg8.Jpeg8Error: Maximum supported image dimension is 65500 pixels

Process finished with exit code 1

2. now I am trying it with jpeg2000 with the same code as above and it´s writing but I don´t know the output yet (I guess size will be more than 3 GB probably).
Code:
img = tifffile.imread(input_C_10)
with TiffWriter(str(output_C_10)+'compressed_10000.tif') as tif:
    tif.write(img, compression='jpeg2000')

I want to ask is this is a correct way to compress or write an image.

**Comment by cgohlke** [comment]
Can you please provide a short, self contained example to reproduce the issue? 36608x186496 is too large for JPEG and untiled JPEG2000, at least when using the libjpeg-turbo and openjpeg libraries. JPEG2000 compression is not part of the TIFF specification. Few software can read such files correctly.

**Comment by Yuvi-416** [comment]
I was able to visualize the same image on ImageJ which was showing black on GNU image manipulation program. I think GNU program didn't support JPEG2000 compressed tiff file. 
I have two questions to ask with you.
First question:
Somehow I finally compress image shape = (36608, 186496) with size 6.83 Gb using jpeg2000 and now the final image size is 3.82GB (still it's big).
I used the below code.
img = tifffile.imread(input_C_10)
with TiffWriter(str(output_C_10)+'compressed_10000.tif') as tif:
tif.write(img, compression='jpeg2000')

Is there any suggestion which I can try to reduce the size, as my final target is to convert it into Dicom (I think I am only allowed to compress this kind of image in lossless jpeg or jpeg2000)?

Second question:
I also use Ndpitools (https://www.imnc.in2p3.fr/pagesperso/deroulers/software/ndpitools/ndpi2tiff.html) for the conversion of ndpi to tiff, although I didn't get the desired result per channel, but I can see that images are compressed using jpeg compression. 
"By default ndpi2tiff will compress data according to the value of the Compression tag found in the source file, that is, JPEG."
How is that? As the image was with the same shape (36608, 186496) but in ndpi format. 
Whenever I try to compress the same image with jpeg i get the following error as,
Traceback (most recent call last):
File "/home/yuvi/PycharmProjects/task-1/VIdeo_frame_extraction/NDPI-ext/TEMP-1-COMPRESSION.py", line 20, in
tif.write(img, compression='jpeg')
File "/home/yuvi/anaconda3/envs/task-1/lib/python3.7/site-packages/tifffile/tifffile.py", line 2340, in write
strip = compress(strip)
File "/home/yuvi/anaconda3/envs/task-1/lib/python3.7/site-packages/imagecodecs/imagecodecs.py", line 829, in jpeg_encode
optimize=optimize, smoothing=smoothing, out=out)
File "imagecodecs/_jpeg8.pyx", line 211, in imagecodecs._jpeg8.jpeg8_encode
imagecodecs._jpeg8.Jpeg8Error: Maximum supported image dimension is 65500 pixels

Is there anything I can do to compress this kind of [(36608, 186496) with size 6.83 Gb] image with jpeg?. How do Ndpitool can manage to compress it with Jpeg?.

Have a nice day.
